pkg/kvcache/kvevents/pool.go

```diff
@@ -0,0 +1,260 @@
+package kvevents
+
+import (
+	"context"
+	"hash/fnv"
+	"sync"
+
+	"github.com/vmihailenco/msgpack/v5"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog/v2"
+
+	"github.com/llm-d/llm-d-kv-cache-manager/pkg/kvcache/kvblock"
+	"github.com/llm-d/llm-d-kv-cache-manager/pkg/utils"
+	"github.com/llm-d/llm-d-kv-cache-manager/pkg/utils/logging"
+)
+
+// Config holds the configuration for the event processing pool.
+type Config struct {
+	// ZMQEndpoint is the ZMQ address to connect to (e.g., "tcp://indexer:5557").
+	ZMQEndpoint string
+	// TopicFilter is the ZMQ subscription filter (e.g., "kv.").
+	TopicFilter string
+	// Concurrency is the number of parallel workers to run.
+	Concurrency int
+}
+
+// DefaultConfig returns a default configuration for the event processing pool.
+func DefaultConfig() *Config {
+	return &Config{
+		ZMQEndpoint: "tcp://indexer:5557",
+		TopicFilter: "kv@",
+		Concurrency: 4,
+	}
```

Comment:
nit: consider making these constants?
They seem to be used in the examples as well, so you may want to import from here.